src/errors/types.rs

```diff
@@ -159,6 +159,11 @@ pub enum ErrorType {
         pattern: String,
     },
     // ---------------------
+    // enum errors
+    Enum {
+        options: String,
```

Comment:
Once merged, I suppose this will be used to replace https://github.com/pydantic/pydantic/blob/b685d64c38f1d7abb3ed37bd30710b3e420dc86a/pydantic/_internal/_std_types_schema.py#L83 with a `PydanticKnownError`?

Is there a reason why you chose `options` here instead of `expected` from that line? I see `expected` is also used in other places in this file.

Reply:
Didn't know about this.

Thanks for mentioning this. I've changed them to `expected`